docs/eigenlayer/operator-guides/operator-installation.md

```diff
@@ -353,7 +353,40 @@ One series of designs involves passing a unique signature from the Operator to t
 
 If the delegationApprover themself calls the DelegationManager.delegateToBySignature function, then they need to provide a [signature from the Restaker](https://github.com/Layr-Labs/eigenlayer-contracts/blob/mainnet/src/contracts/core/DelegationManager.sol#L157-L204). The approverSignatureAndExpiry input is ignored if the caller is themselves the delegationApprover. One potential drawback to this approach is the delegationApprover would pay the gas for the transaction.
 
-**Whitelisting and Blacklisting Restakers for Delegation**
+#### Generating approval signatures using eigenlayer-cli
+If you want to generate signatures for stakers using delegationApprover address, you can use eigenlayer-cli (>= v0.10.8) to generate those. Use the below command to generate
+```bash
+eigenlayer operator get-delegation-approval \
+  --ecdsa-private-key <delegation-approval-address-private-key> \
+  operator.yaml <staker-address>
+  --
+```
+It will generate a signature like below
+```bash
+operator: 0x2222AAC0C980Cc029624b7ff55B88Bc6F63C538f
+approverSignatureAndExpiry.signature: 0xd8af4e2d294d644a989a517583420037d9a089de23bb828b3c00e309e5c6517b236221a5af145cea9eeba59f24732bb410efa79bc840130724b2bf23640011271c
+approverSignatureAndExpiry.expiry: 1729989609
+approverSalt: 0xdca4f1809aeb9c0f7059e328d1e28b317efff44b4ae9c2de67a53af8865876d3
+```
+You are provide these details to stakers and they would be able to successfully delegate to the operator. By default, the expiry of this salt is `3600` seconds. In order to change it, use the `--expiry` flag to provide the desited value. You can also use `--path-to-key-store` flag instead of `--ecdsa-private-key` if you have your approval key as a keystore. We do NOT support web3signer or fireblocks for this operation. 
```

Comment:
```suggestion
Provide these details to stakers so they can successfully delegate to the operator. By default, the salt’s expiry is set to 3600 seconds. To modify this, use the --expiry flag followed by the desired expiry value. You can also use `--path-to-key-store` flag instead of `--ecdsa-private-key` if you have your approval key as a keystore. We do NOT support web3signer or fireblocks for this operation. 
```